### ISSUE TITLE
Random Integer Key

### ISSUE THREAD

**Issue by nerun** [mandeep/Mausoleum#9]
When Random Integer Key is checked, key is not created:
```
$ mausoleum-gui 
tomb  .  Commanded to dig tomb 
tomb (*) Creating a new tomb in teste.tomb
tomb  .  Generating teste.tomb of 10MiB
10+0 registros de entrada
10+0 registros de saída
10485760 bytes (10 MB, 10 MiB) copiados, 0,0477045 s, 220 MB/s
-rw------- 1 daniel daniel 10M jan 13 23:06 teste.tomb
tomb (*) Done digging teste.tomb
tomb  .  Your tomb is not yet ready, you need to forge a key and lock it:
tomb  .  tomb forge teste.tomb.key
tomb  .  tomb lock teste.tomb -k teste.tomb.key
tomb [E] Unrecognized option --use-urandom for subcommand forge
tomb [W] File is not yet a tomb: teste.tomb
tomb  .  Valid tomb file found: teste.tomb
tomb  .  Commanded to lock tomb teste.tomb
tomb  .  Checking if the tomb is empty (we never step on somebody else's bones).
tomb  .  Fine, this tomb seems empty.
tomb [E] Key not found, specify one using -k.
```
But everything else works fine. Of course, just if fix for issue #8 is applied.

**Comment by mandeep** [comment]
Seems like this option needs to be changed. Not sure what the new option is so I'll have to look into it.

**Comment by mandeep** [comment]
fixed by 440493b204c0d671dfc08262a6fb96db8c969079
